pom.client.xml

```diff
@@ -861,6 +861,53 @@
         <module>./sdk/template/azure-sdk-template</module>
       </modules>
     </profile>
+
+    <!-- Skip overview generation unless -Dgenerate-overview is passed as an argument.
```

Comment:
Is there any reason this is in both client and data as opposed to in the shared parent? Not a huge deal but wondering if there was a reason.

Reply:
The primary reason was that I don't want to inadvertently make any changes that'll affect Management builds.